src/Chapter2.hs

```diff
@@ -534,6 +555,7 @@ For example, we can patch the previous function to count the number of
 steps we need to take in order to reduce the number to zero.
 
 🤔 Blitz question: can you guess what this number represents?
+The size of the stack created by the recursive function
```

Comment:
Yes, you are on the right track. But the question was more about what is the meaning of the number of such recursive steps? Notice, that on each step you divide the original number by 2 each time. Try to think, what number you will get in the end, and how it's connected to the given number. Because you will get different results for different inputs